lib/spells/CSpellHandler.cpp

```diff
@@ -551,6 +532,59 @@ ESpellCastProblem::ESpellCastProblem CSpell::isImmuneByStack(const CGHeroInstanc
 	return ESpellCastProblem::OK;
 }
 
+void CSpell::prepareBattleLog(const CBattleInfoCallback * cb,  const BattleSpellCast * packet, std::vector<std::string> & logLines) const
+{
+	bool displayDamage = true;
+	
+	std::string casterName("Something"); //todo: localize
+	
+	if(packet->castedByHero)
```

Comment:
Past form of "cast" is still "cast" not "casted", just FYI ;) http://www.usingenglish.com/reference/irregular-verbs/cast.html